README.md

```diff
@@ -12,12 +12,4 @@ VSB-TUO is deployed at `https://kelvin.cs.vsb.cz`.
 If you find any bugs in Kelvin or want to suggest new features and improvements, please file an
 [issue](https://github.com/mrlvsb/kelvin/issues/new).
 
-You can find documentation on how Kelvin works and how to contribute to it in the following files:
-- [Architecture](docs/architecture.md) describes the high-level architecture of Kelvin.
-- [First-time setup](docs/setup.md) provides a guide on how to configure Kelvin for local development.
-- [Development](docs/development.md) contains useful hints on how to perform various
-  development tasks.
-- [Deployment](docs/deployment.md) describes how is Kelvin deployed to a production environment.
-
-You can find user documentation in the following files:
-- [Teacher's guide](docs/teacher-guide.md) contains useful information for teachers using Kelvin.
+You can find documentation of Kelvin's internal workings and how to contribute in [docs](https://mrlvsb.github.io/kelvin/).
```

Comment:
Could you please add a command here that shows how to build the documentation locally?